supervisor/api/resolution.py

```diff
@@ -62,3 +78,17 @@ async def dismiss_issue(self, request: web.Request) -> None:
     def healthcheck(self, request: web.Request) -> Awaitable[None]:
         """Run backend healthcheck."""
         return asyncio.shield(self.sys_resolution.healthcheck())
+
+    @api_process
+    @require_home_assistant
+    async def check_options(self, request: web.Request) -> None:
```

Comment:
Since the word `check` is ambiguous, could we rename this to `set_check_options` ?

Reply:
sure, but since @pvizeli marked it as draft I will not be doing anything until he is done commenting